fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/impl/IndexBuilderImpl.java

```diff
@@ -63,7 +70,12 @@ public void rebuild() {
 
         LOGGER.info("Initiating index rebuild.");
         fedoraToOCFLObjectIndex.reset();
+        final var transaction = transactionManager.create();
+        final var txId = transaction.getId();
+        LOGGER.info("Resetting ContaintmentIndex...");
```

Comment:
The `fedoraToOCFLObjectIndex.reset();` clears the index and deletes the file on disk if is exists.

Reply:
Okay, `LOGGER.info("Resetting ContaintmentIndex...");` should probably happen before `fedoraToOCFLObjectIndex.reset();` in that case, otherwise its coming a little on the late side :)

Reply:
Ohhhhh sorry I was looking at the top `LOGGER.info("Initiating index rebuild.");`, on the main page it didn't show the below LOGGER.info().

Please disregard this comment, it appears that both the fedoraToOcflIndex and containment index is rebuilt at the same time. So additional log statements are probably not necessary.

Reply:
Removed useless log messages.